docs/roadmap.markdown

```diff
@@ -4,9 +4,122 @@ title: "Roadmap"
+
+   * Built on the New Relic Client
+   * Fully implementations APIs available through the New Relic Client
+   <br/>
+
+* **New Relic Client** - The New Relic Client provides the building blocks for many tools in the toolkit, enabling quick access to the suite of New Relic APIs.  As a library, it can also be leveraged within your own custom applications.
+
+   * Full support for the followin New Relic API endpoints
+     * APM REST v2 API
+     * Infrastructure API
+     * Synthetics API
+     * GraphQL
+   <br/>
+
+* **New Relic CLI** - The New Relic CLI allows integrating New Relic into your existing workflows. Be it fetching data from your laptop while troubleshooting an issue, or adding New Relic into your CI/CD pipeline.
+
+   * Build on the New Relic Client
+   * Initial implementation of
+     * Deployment Markers
+     * Tagging
+   <br/>
+
+* **[New Relic AWS CloudFormation Integration](https://github.com/newrelic/cloudformation-partner-integration)** - The AWS CloudFormation Integration enables developers using CloudFormation to easily provision a subset of New Relic resources.
+
+   * Build on the NR Client
+   * Initial implementation of
+     * Basic NRQL Alert configuration
+   <br/>
+
+
+## Milestones
+
+Ongoing work will be tracked in the [Developer Toolkit](https://github.com/orgs/newrelic/projects/6) Project, which will link to each individual tool that is in flight.  Here the latest summary for planned work by the New Relic team:
+
+### 2019 Winter
+
+| State | Tool | Milestone | Rationale |
+| ----- | ---- | --------- | --------- |
+| Completed | New Relic Terraform Provider | (Tracked Internally)| The current Terraform Provider for New Relic was created by the community, and suffering from lack of attention.  DTK will integrate into the ecosystem, and create a clean base to work from. |
+| **Current** | New Relic Terraform Provider | [Feature Parity with v2 APIs](https://github.com/terraform-providers/terraform-provider-newrelic/projects/1?card_filter_query=milestone%3A%22feature+parity+with+v2+apis%22) | Enhance the Terraform provider to include all capabilities of the REST v2 API and NerdGraph data access, including configuring Cloud Integrations and Tagging |
+| Next | New Relic Client | [New Relic Client Consolidation](https://github.com/newrelic/newrelic-client-go/projects/1?card_filter_query=milestone%3A%22new+relic+client+consolidation%22) | Currently there is not a consistent, easy to use experience for developing against the New Relic APIs.  As the Developer Toolkit team continues to create new tools for users, having a consolidated library.  This MMF strives to create a unified client library for consumption by the community, and specifically utilized in the New Relic Terraform Provider. |
```

Comment:
Change to Client Library?  Also, remove reference to Developer Toolkit team and replace with We or New Relic.  The sentence is also incomplete - "As we continue to create new tools for users, having a consolidated library" - will do what?